exporters/committime/collector_base.py

```diff
@@ -90,54 +103,73 @@ def generate_metrics(self) -> Iterable[CommitMetric]:
         # Initialize metrics list
         metrics = []
         for namespace in self._namespaces:
-            # Initialized variables
-            builds = []
-            apps = []
-            builds_by_app = {}
             app_label = pelorus.get_app_label()
             logging.debug(
                 "Searching for builds with label: %s in namespace: %s"
                 % (app_label, namespace)
             )
-
+            # This get all Builds in our cluster
             v1_builds = self._kube_client.resources.get(
                 api_version="build.openshift.io/v1", kind="Build"
             )
-            # only use builds that have the app label
+            # only use builds that have an app label
             builds = v1_builds.get(namespace=namespace, label_selector=app_label)
 
-            # use a jsonpath expression to find all values for the app label
-            jsonpath_str = (
-                "$['items'][*]['metadata']['labels']['" + str(app_label) + "']"
-            )
+            try:
+                v1_tekton = self._kube_client.resources.get(
+                    api_version="tekton.dev/v1beta1", kind="PipelineRun"
+                )
+                # only use PipelineRun that have an app label
+                pipeline_runs = v1_tekton.get(
+                    namespace=namespace, label_selector=app_label
+                )
+            except openshift.dynamic.exceptions.ResourceNotFoundError:
+                pipeline_runs = pelorus.NoOpResourceInstance()
+
+            # use a jsonpath expression to find all possible values for the app label
+            jsonpath_str = f"$['items'][*]['metadata']['labels']['{app_label}']"
```

Comment:
This will break if app_label has single quotes in it for some reason. We could extract this part with plain python.